modules/porous_flow/doc/content/modules/porous_flow/tests/capillary_pressure/capillary_pressure_tests.md

```diff
@@ -1,3 +1,35 @@
 # Capillary pressure test descriptions
 
-TODO
+#### The detail documentation of the implementation of this code packet can be founded in [here](porous_flow/capillary_pressure.md).
+
+## Brooks-Corey
+
+!media capillary_pressure/brook_corey.png style=width:100%;margin-left:10px; caption=Brooks-Corey Capillary Pressure test cases id=brookcoreyPc
+
+The input file for log-extension off test:
```

Comment:
Can you reword this to say that there are two test cases - with and without the log extension at low saturation's